spec/serializers/organization_serializer_spec.rb

```diff
@@ -88,4 +88,22 @@
       end
     end
   end
+
+  describe "#urls" do
+    let(:result) do
+      OrganizationSerializer.single({}, Organization.where(id: organization.id), {})
```

Comment:
it's not well documented from memory, basically the same that a page takes the params hash, the AR scope and the serializer context hash. 

From https://github.com/RestPack/restpack_serializer#exposing-an-api
> The AlbumSerializer also provides a single method which will return a serialized resource similar to as_json above.

https://github.com/RestPack/restpack_serializer/blob/8eb3096c7118968ab6f353b09b1e6976a6c36951/lib/restpack_serializer/serializable/single.rb#L5